build/grunt.js

```diff
@@ -190,6 +190,7 @@ module.exports = function(grunt) {
       swf:   { cwd: 'node_modules/videojs-swf/dist/', src: 'video-js.swf', dest: 'build/temp/', expand: true, filter: 'isFile' },
       ie8:   { cwd: 'node_modules/videojs-ie8/dist/', src: ['**/**'], dest: 'build/temp/ie8/', expand: true, filter: 'isFile' },
       dist:  { cwd: 'build/temp/', src: ['**/**', '!test*'], dest: 'dist/', expand: true, filter: 'isFile' },
+      a11y:  { src: 'sandbox/descriptions.html.example', dest: 'sandbox/descriptions.test-a11y.html' }, // Can only test a file with a .html or .htm extension
```

Comment:
Everything inside of `sandbox` is ignored. The idea is that you can do whatever you want in there and it won't be accidentally checked in. The issue arrises if we have a `index.html` file inside of `sandbox` (without the `.example`), when you make changes to that file, even though git is supposed to ignore `sandbox/` because `index.html` was previously checked in, it'll show up as modified. Then, if people were trying to make a PR, modified `sandbox/index.html` and committed by running `git add -A` the change to the `sandbox/index.html` would get added to the commit.

Maybe, we can move the `.example` files into something like `build/examples` and then have a grunt task that copies those files into the sandbox? We can even do it as a post-install (when you're doing `npm install` on a clone, for example).